ma_search/config.py

```diff
@@ -0,0 +1,132 @@
+"""
+MetAlert Search : Main Config
+=============================
+
+Copyright 2021 MET Norway
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+import yaml
+import logging
+
+logger = logging.getLogger(__name__)
+
+class Config():
+    """Main config class wrapping the config yaml file.
+    """
+
+    def __init__(self):
+
+        # Internals
+        self._pkgRoot = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+        self._rawConf = {}
+
+        # Config Settings
+        # ===============
+        # These should only be set from file.
+
+        # Core Values
+        self.dbProvider = None
+
+        # SQLite Settings
+        self.sqlitePath = None
+
+        return
+
+    def readConfig(self, configFile=None):
+        """Read the config file.
+
+        Parameters
+        ----------
+        configFile : str or None, optional
+            The config file to be loaded. If None, pgkRoot/config.yaml
+            is attempted loaded instead.
+
+        Returns
+        -------
+        bool
+            True of successful, False if anything failed.
```

Comment:
``` suggestion
True if successful, False if anything failed.
```

Reply:
Ok, fixed locally. Will commit when the other one is resolved.